cmd/vulcan-trivy/vulcan-trivy.go

```diff
@@ -24,23 +26,41 @@ import (
 	check "github.com/adevinta/vulcan-check-sdk"
 	"github.com/adevinta/vulcan-check-sdk/helpers"
 	checkstate "github.com/adevinta/vulcan-check-sdk/state"
+
 	"github.com/avast/retry-go"
 )
 
-const vulnCVETruncateLimit = 10
+const (
+	vulnCVETruncateLimit = 10
+	DefaultDepth         = 1
+)
 
 var (
 	checkName        = "vulcan-trivy"
 	logger           = check.NewCheckLog(checkName)
 	reportOutputFile = "report.json"
+	localTargets     = regexp.MustCompile(`https?://(localhost|host\.docker\.internal|172\.17\.0\.1)`)
+
+	FilePatters = []string{
+		// trivy only detect requirements.txt files
+		`pip:/requirements/[^/]+\.txt`,    // All the .txt files in a requirements directory.
+		`pip:[^/]*requirements[^/]*\.txt`, // All the files .txt that contains requirements
+	}
 )
 
 type options struct {
-	ForceUpdateDB bool   `json:"force_update_db"`
-	IgnoreUnfixed bool   `json:"ignore_unfixed"`
-	Severities    string `json:"severities"`
+	ForceUpdateDB  bool   `json:"force_update_db"`
+	IgnoreUnfixed  bool   `json:"ignore_unfixed"`
+	Severities     string `json:"severities"`
+	Depth          int    `json:"depth"`
+	Branch         string `json:"branch"`
+	FSEnabled      bool   `json:"fs_enabled"`
+	ImageEnabled   bool   `json:"image_enabled"`
```

Comment:
A bit nitpicky, but just to be consistent I would also pluralize this field to "images_enabled" as with the rest.

Reply:
In fact it is a good point, I refactored it to make clear the map with trivy options and allow all the combinations.

```sh
trivy fs --security-checks vuln,secret,config
trivy image --security-checks vuln,secret,config
```

Also I moved the `config` scan into an  `--security-checks` option.

```sh
trivy config
```

Reply:
I love it! 👌